src/Main.php

```diff
@@ -0,0 +1,211 @@
+<?php
+namespace Tribe\Extensions\Attendee_Data_Editor;
+
+use Tribe__Tickets_Plus__Tickets_View as Tickets_View;
+
+class Main {
+	private $plugin_url = '';
+	private $assets_enqueued = 0;
+
+	public function __construct( $plugin_url ) {
+		$this->plugin_url = $plugin_url;
+
+		add_action( 'wp_ajax_load_attendee_data_editor_form',[ $this, 'editor_content' ] );
+		add_action( 'wp_ajax_save_attendee_data_editor_form', [ $this, 'save_fields' ] );
+		add_action( 'admin_init', [ $this, 'add_assets' ] );
+		add_filter( 'tribe_events_tickets_attendees_table_column', [ $this, 'add_edit_link' ], 20, 3 );
+	}
+
+	public function add_assets() {
+		// Does this look like an attendee screen?
+		if ( empty( $_GET['page'] ) || 'tickets-attendees' !== $_GET['page'] ) {
+			return;
+		}
+
+		// Execute once only
+		if ( $this->assets_enqueued++ ) {
+			return;
+		}
+
+		$this->assets();
+	}
+
+	protected function assets() {
+		wp_enqueue_style(
+			'attendee-data-editor-styles',
+			$this->plugin_url . 'src/resources/css/attendee-data-editor.css'
+		);
+
+		wp_enqueue_script(
+			'attendee-data-editor-script',
+			$this->plugin_url . 'src/resources/js/attendee-data-editor.js',
+			[ 'jquery' ],
+			false,
+			true
+		);
+
+		wp_localize_script(
+			'attendee-data-editor-script',
+			'attendee_data_editor', [
+				'check' => wp_create_nonce( 'attendee-data-editor' ),
+				'save_btn' => __( 'Save!', 'tribe-ext-attendee-data-editor' ),
+				'oh_oh_on_save' => __( 'Something went wrong: your attendee data could not be saved.', 'tribe-ext-attendee-data-editor' ),
```

Comment:
"uh_oh" or did you mean "oh_oh"?

Reply:
I would say 'Oh Oh!' but in the interests of internationally friendly code, I'll switch to 'Uh Oh!'